packages/builder/src/builder.js

```diff
@@ -497,6 +498,18 @@ export default class Builder {
       }
     }))
 
+    // Check styleResource exist
```

Comment:
Maybe [`options.js`](https://github.com/nuxt/nuxt.js/blob/dev/packages/common/src/options.js) is a better place for the check?

Reply:
Maybe. But why check for plugins.js is here and not in options then  ? :)

Reply:
Wecare trying to scan file system which is only specific to builder ( l mean not required for production nuxt start) So here is probably better place and helps decoupling packages.